Content.Server/GlobalVerbs/ControlMobVerb.cs

```diff
@@ -14,19 +16,22 @@ public class ControlMobVerb : GlobalVerb
     {
         public override bool RequireInteractionRange => false;
 
+        private bool TransferringToVisited([CanBeNull] Mind userMind, IEntity target)
```

Comment:
Instead of using `[CanBeNull]` etc you can stick `#nullable enable` at the top of the file and then use C# 8 nullable reference types (e.g. `Mind?`)